code/obj/stool.dm

```diff
@@ -1097,6 +1099,116 @@
 				src.arm_image.layer = FLY_LAYER+1
 				src.UpdateOverlays(src.arm_image, "arm")
 
+	attackby(obj/item/W as obj, mob/user as mob)
+		if (iswrenchingtool(W) && status==1)
+			user.visible_message("<b>[user]</b> begins removing the center supports of the pew.")
+			playsound(src.loc, "sound/items/Ratchet.ogg", 50, 1)
+			actions.start(new /datum/action/bar/icon/furniture_deconstruct(src, W, 30), user)
+		if (isscrewingtool(W))
+			playsound(src.loc, "sound/items/Screwdriver.ogg", 50, 1)
+			switch(status)
+				if(3)
+					realign()
+					user.visible_message("<b>[user]</b> adjusts the pew.")
+					return
+				if(2)
+					status = 1
+					anchored = 0
+					user.visible_message("<b>[user]</b> unscrews the end piece fasteners of the pew and unsecures it from the floor.")
+					return
+				if(1)
+					status = 2
+					anchored = 1
+					user.visible_message("<b>[user]</b> screws in the end piece fasteners of the pew and secures it to the floor.")
+					return
+		if (ispryingtool(W))
+			if(status!=1)
+				playsound(src.loc, "sound/items/Crowbar.ogg", 50, 1)
+			switch(status)
+				if(3)
+					status = 2
+					user.visible_message("<b>[user]</b> pries off the outer pew cover.")
+					if(prob(10))
+						user.visible_message(pick("This isn't how pews are supposed to work.","This seems highly unlikely","You can almost feel a figure judging you for destroying church pews.","You can almost make out some writing under the cover... <i>Fenrik was here</i>"))
+					return
+				if(2)
+					status = 3
+					user.visible_message("<b>[user]</b> pries the pew cover back on.")
+					return
+				if(1)
+					src.dir = turn(src.dir, -90)
+					return
+
+	proc/realign()
+		icon_state = "pew"
+		switch(src.dir)
+			if(NORTH)
+				for (var/obj/stool/chair/pew/P in get_step(src, EAST))
+					if(P.dir==src.dir)
+						icon_state = "pewL"
```

Comment:
doing icon state stuff like this is pretty ugly, due to having the fancy pew subtype that can't reuse any of this code and results in massive copy-pasting